staging/src/k8s.io/component-base/metrics/opts_test.go

```diff
@@ -208,3 +208,21 @@ metric2,label2: v3`,
 		})
 	}
 }
+
+func TestResetLabelValueAllowLists(t *testing.T) {
+	labelValueAllowLists = map[string]*MetricLabelAllowList{
+		"metric1": {
+			labelToAllowList: map[string]sets.Set[string]{
+				"label1": sets.New[string]("v1", "v2"),
+			},
+		},
+		"metric2": {
+			labelToAllowList: map[string]sets.Set[string]{
+				"label2": sets.New[string]("v3"),
+			},
+		},
+	}
+
+	ResetLabelValueAllowLists()
+	assert.Equal(t, map[string]*MetricLabelAllowList{}, labelValueAllowLists)
```

Comment:
you should be able to use `assert.Empty` here

Reply:
Updated. Thanks